template.yaml

```diff
@@ -41,10 +41,6 @@ Parameters:
     Type: String
     Description: Reference to index containing publications by owner
     Default: ByPublisher
-  PublicationsDoiRequestsByStatusIndexName:
```

Comment:
Removed on purpose?

Reply:
Linter says it is not used by anything, I could not see it in use either.